.aspect/workflows/terraform/terraform.bzl

```diff
@@ -0,0 +1,20 @@
+"""Wrapper around a terraform sh_binary target"""
+
+def terraform(name, **kwargs):
```

Comment:
IMO there's too much code here just for the purpose of pinning the terraform version. We like to do that when we're all-out crazy about reproducibility, but that's not what other terraform users do. I think we should keep our OSS footprints small, as there's enough code here that it's going to become a maintenance burden to manage many copies.

Possible alternative is to create/find&contribute a rules_terraform to fix the duplication problem, but my suggestion is we just run `terraform` like "normal engineers" in our OSS repos.

Reply:
The lock file may jump around depending on who's running the terraform apply. Personally, I'm very used to running `bazel run terraform` and don't really want to worry about what version I've got install locally or if I even have one installed. Maybe a rules_terraform is the way to go here so we can re-use across our OSS repos for all the deployments and use it internally for silo as well.

Reply:
I'll leave it out for now so this can land with less code but will think about an aspect-build/rules_terraform.

Reply:
you can just `alias terraform=/absolute/path/to/silo/bazel-out/some/path/terraform` right? Is that enough solution for now to get the terraform binary you want for yourself?

Reply:
suppose it is for now so I don't have to go make a new rule set :)